FunnyFridays/FunnyFridays.WebClient/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,50 @@
+@{
+	ViewData["Title"] = "Home Page";
+}
+
+<app asp-prerender-module="ClientApp/dist/main-server">Loading...</app>
+
+@section scripts {
+	<script src="~/dist/vendor.js" asp-append-version="true"></script>
+	<script src="~/dist/main-client.js" asp-append-version="true"></script>
+}

```

Comment:
i think "map" functionality should be in separate view, not in Index

Reply:
https://github.com/angular/angular/issues/4903